test/test_slave.html

```diff
@@ -18,7 +18,13 @@
   <head>
     <title>PDF.js test slave</title>
     <meta charset="utf-8">
-    <script src="../build/generic/build/pdf.js"></script>
+    <script>
+    if (location.search.replace('?', '&').includes('&coverage=true')) {
+      document.write('<script src="../coverage/build/generic/build/pdf.js"></script>');
+    } else {
+    document.write('<script src="../build/generic/build/pdf.js"></script>');
```

Comment:
The indentation is still incorrect. This `document.write` line is still at the same indentation level as the `} else {` line.